pdc/apps/release/templates/product_list_include.html

```diff
@@ -5,10 +5,10 @@
     <th>{% trans "Short Name" %}</th>
     <th>{% trans "Name" %}</th>
     <th>
-        <select id="active_filter">
-            <option value="All">Active-ALL</option>
-            <option value="True">True</option>
-            <option value="False">False</option>
+        <select id="active_filter" value= {{paginator.per_page}}>
```

Comment:
@lubomir; it don't work well when we apply `data-per-page`, because the Datatables will override the pagination, and set the page_size as 20.